tests/system/action/motion_workflow/test_delete.py

```diff
@@ -40,6 +40,28 @@ def test_delete_with_states(self) -> None:
         self.assert_model_deleted("motion_workflow/2")
         self.assert_model_deleted("motion_state/3")
 
+    def test_delete_with_first_state(self) -> None:
+        self.create_model(
+            "meeting/1",
+            {"motion_workflow_ids": [2, 100]},
+        )
+        self.create_model(
+            "motion_workflow/2",
+            {"meeting_id": 1, "state_ids": [3], "first_state_id": 3},
+        )
+        self.create_model(
+            "motion_state/3", {"workflow_id": 2, "first_state_of_workflow_id": 2}
+        )
+        # needed because you can't delete the last workflow of a meeting
+        self.create_model("motion_workflow/100", {"meeting_id": 1})
+        response = self.client.post(
+            "/",
+            json=[{"action": "motion_workflow.delete", "data": [{"id": 2}]}],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_deleted("motion_workflow/2")
+        self.assert_model_deleted("motion_state/3")
+
```

Comment:
This was the key point why the test failed, I extracted it into it's own test to not have to load all example data.

Reply:
Yes. We can leave the `load_example_data` in code for such testing cases but should not use it in permanent tests because it is too slow.